unittests/test_tags.py

```diff
@@ -160,13 +160,7 @@ def test_finding_create_tags_with_commas(self):
         finding_id = self.create_finding_with_tags(tags)
         response = self.get_finding_tags_api(finding_id)
 
-        # the old django-tagging library was splitting this tag into 2 tags
-        # with djangotagulous the tag does no longer get split up and we cannot modify tagulous
-        # to keep doing the old behaviour. so this is a small incompatibility, but only for
-        # tags with commas, so should be minor trouble
-        #
-        # self.assertEqual(2, len(response.get('tags')))
-        self.assertEqual(1, len(response.get('tags')))
+        self.assertEqual(2, len(response.get('tags')))
         # print("response['tags']:" + str(response['tags']))
```

Comment:
We should drop this commented out print line

Reply:
Sure will do